frontend/src/services/ChatRequests.js

```diff
@@ -25,6 +25,17 @@ const request = (options) => {
   );
 };
 
+export function getAvailabilitySlots(doctorId) {
+  if (!localStorage.getItem(ACCESS_TOKEN)) {
```

Comment:
Could you please move availability requests to the separate file?

Reply:
Sure. Thanks